card/mod/05_standard/set/abstract/attachment.rb

```diff
@@ -24,15 +24,17 @@ def self.included host_class
 end
 
 event :assign_attachment_on_create, after: :prepare, on: :create, when: proc { |c| c.save_preliminary_upload? } do
-  if (action = Card::Action.fetch(Card::Env.params[:cached_upload]))
+  if (action = Card::Action.fetch(@cached_upload))
     upload_cache_card.selected_action_id = action.id
     upload_cache_card.select_file_revision
     assign_attachment upload_cache_card.attachment.file, action.comment
   end
 end
 
-event :assign_attachment_on_update, after: :prepare, on: :update, when: proc { |c| c.save_preliminary_upload? } do
-  if (action = Card::Action.fetch(Card::Env.params[:cached_upload]))
+event :assign_attachment_on_update, after: :prepare, on: :update, when:  proc {
+ |c| c.save_preliminary_upload? 
```

Comment:
Block argument expression is not on the same line as the block start.<br>Trailing whitespace detected.